internal/infra/postgres/dsn.go

```diff
@@ -0,0 +1,62 @@
+package postgres
+
+import (
+	"bytes"
+	"fmt"
+	"time"
+)
+
+const (
+	defaultLocation       = "Asia/Tokyo"
+	disabledSslModeString = "disable"
+)
+
+type DSNConfig struct {
+	Host     string
+	User     string
+	Password string
+	DbName   string
+	Port     int
+	SslMode  bool
+	Loc      *time.Location
```

Comment:
頭文字とった略語は大文字・小文字そろえるという Go の命名規則があります

```suggestion
	DBName   string
	Port     int
	SSLlMode  bool
	Loc      *time.Location
```